tests/test_generate_readme

```diff
@@ -0,0 +1,15 @@
+"""This module tests the generate_readme module."""
+
+import os
+
+from configator import create_readme
+
+def test_create_readme(mocker):
+    """tests create_gatorgrader function, specifically
+    gatorgrader.yml containing fastfail condition.
+    """
+    mock_open = mocker.mock_open()
+    mocker.patch('builtins.open', mock_open)
+    mocker.patch('os.mkdir')
+    create_gatorgradle_yml.create_gatorgrader()
```

Comment:
This is not the function under test. `generate_readme.create_readme()` should be invoked here.